internal/transactionsubmission/manager_test.go

```diff
@@ -142,9 +128,27 @@ func Test_SubmitterOptions_validate(t *testing.T) {
 		},
 		{
 			name: "🎉 successfully finishes validation with nil crash tracker client",
+			submitterOptions: SubmitterOptions{
+				DatabaseDSN: dbt.DSN,
+				MonitorService: tssMonitor.TSSMonitorService{
+					Client:        &monitorMocks.MockMonitorClient{},
+					GitCommitHash: "0xABC",
+					Version:       "0.01",
+				},
+				HorizonURL:           "https://horizon-testnet.stellar.org",
+				NetworkPassphrase:    network.TestNetworkPassphrase,
+				PrivateKeyEncrypter:  &utils.PrivateKeyEncrypterMock{},
+				DistributionSeed:     "SBDBQFZIIZ53A7JC2X23LSQLI5RTKV5YWDRT33YXW5LRMPKRSJYXS2EW",
+				NumChannelAccounts:   1,
+				QueuePollingInterval: 10,
+				MaxBaseFee:           txnbuild.MinBaseFee,
+			},
+		},
+		{
+			name: "validate monitorService",
 			submitterOptions: SubmitterOptions{
 				DatabaseDSN:          dbt.DSN,
-				MonitorService:       &monitor.MonitorService{},
+				MonitorService:       tssMonitor.TSSMonitorService{},
```

Comment:
Just for the sake of organization, can you move this assertion right above `🎉 successfully finishes validation with nil crash tracker client`?